launch.go

```diff
@@ -506,6 +493,26 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 		delete(metadata.(map[string]interface{}), "summary")
 	}
 
+	// Set build parameter explicitly (Issue #2501)
+	if build.Meta["parameters"] != nil {
+		mergedMeta["parameters"] = build.Meta["parameters"]
+	}
+
+	buildMeta := map[string]interface{}{
+		"pipelineId": strconv.Itoa(job.PipelineID),
+		"eventId":    strconv.Itoa(build.EventID),
+		"jobId":      strconv.Itoa(job.ID),
+		"buildId":    strconv.Itoa(buildID),
+		"jobName":    job.Name,
+		"sha":        build.SHA,
+	}
+
+	if coverageErr == nil {
+		buildMeta["coverageKey"] = coverageInfo.EnvVars["SD_SONAR_PROJECT_KEY"]
+	}
+
+	mergedMeta["build"] = buildMeta
```

Comment:
@yoshwata  There is an issue with this change. meta data set on `build` key is now lost because it's overwriting `build` key

https://cd.screwdriver.cd/pipelines/7217/builds/721794/steps/meta-set
https://cd.screwdriver.cd/pipelines/7217/builds/721795/steps/meta-get

Reply:
OK, thank you for report 🙇